system/hello_double_tap/hello_double_tap.c

```diff
@@ -13,6 +13,9 @@
 // `pico_bootsel_via_double_reset` library!
 
 int main() {
+#ifndef PICO_DEFAULT_LED_PIN
```

Comment:
yup, i seem to have lost my changes for this... can you fix this one up to just not use the LED instead

Reply:
Done.